### ISSUE TITLE
Calistoga font is missing

### ISSUE THREAD

**Issue by JuliaArr** [elementor/elementor#10821]
Hi,

I want to use the Calistoga google font: https://fonts.google.com/specimen/Calistoga

But I cannot find it in the typography settings: https://www.screencast.com/t/3AK0YyKbnzV

How can it be fixed?

Thanks in advance!

**Comment by DeoThemes** [comment]
The same with DM Sans. Been asking this font since July 20, 2019 - https://github.com/elementor/elementor/issues/8637

It's sad too see that Elementor doesn't care of updating new fonts, all other plugins that I use always keep fonts updated.

The only option you have is to use custom code, add it to your functions.php:

```
/**
 * Add Font Group
 */
add_filter( 'elementor/fonts/groups', function( $font_groups ) {
	$font_groups['theme_fonts'] = __( 'Theme Fonts', 'elementorkit' );
	return $font_groups;
} );


/**
 * Add Group Fonts to the fonts control
 */
add_filter( 'elementor/fonts/additional_fonts', function( $additional_fonts ) {
	// Key/value
	//Font name/font group
	$additional_fonts['Calistoga'] = 'theme_fonts';
	return $additional_fonts;
} );

// Now if they are included by your theme, you are done.
// if not you can hook in to the elementor/fonts/print_font_links/{$font_type} action hook to enqueque your theme fonts, ex:
add_action( 'elementor/fonts/print_font_links/theme_fonts', function( $font_name ) {
  wp_enqueue_style( 'my-theme-google-fonts', '//fonts.googleapis.com/css?family=Calistoga:400' );
} );
```

**Comment by cirkut** [comment]
Yep, I'm also looking to use DM Sans. Kinda bogus that they can't spend 15 minutes getting the latest Google Fonts supported.

**Comment by MichaelDarko** [comment]
Same problem with DM Serif Display font. :-/

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0.0

Feel free to update

Thanks!
